### PR TITLE
added snorkel shop as search item for scuba shop

### PR DESCRIPTION
**Pull request by fakeharahman**
Added "Snorkel Shop" as a search term for "Scuba Shop" from issue#7857. 

Added the "Snorkel" term in `terms` in data/presets/presets/shop/scuba_diving.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "abortcontroller-polyfill": "^1.4.0",
     "aes-js": "^3.1.2",
     "alif-toolkit": "^1.2.9",
+    "all": "0.0.0",
```
Comment (quincylvania) [inline package.json]: Looks like you may have accidentally added this dependency? Could you delete this line?

Comment (fakeharahman) [reply package.json]: Thank you! I fixed the error

Comment (quincylvania) [reply package.json]: Thanks! Looks like your text editor changed the indentations? Would you mind reseting the file?

Comment (fakeharahman) [reply package.json]: Sorry about that, just fixed it!

**Comment by quincylvania** [comment]
@fakeharahman Looks great, thanks! Welcome to iD dev

**Pull request merged**